contracts/ics-escrow/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "ics-escrow"
+version = "0.1.0"
+authors = ["callumanderson <callumanderson745@gmail.com>"]
+edition = "2018"
+
+exclude = [
+  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
+  "contract.wasm",
+  "hash.txt",
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[dependencies]
+cosmwasm-std = "1.0.0"
+cosmwasm-storage = "1.0.0"
+cw-storage-plus = "0.13.2"
+cw2 = "0.13.2"
+cw721 = "0.13"
```

Comment:
Going to publish [v0.13.4](https://github.com/CosmWasm/cw-nfts/releases/tag/v0.13.4) as a crate.io release tomorrow (want to sleep on it first). `v0.13.4` has some fixes, and it will also be using the same deps as noted here.